awscli/help.py

```diff
@@ -259,6 +272,39 @@ def event_class(self):
     def name(self):
         return self.obj.name
 
+    @property
+    def topic_table(self):
```

Comment:
Not really a fan of breaking LSP here.  We're adding a property that is specific to a single subclass of the `HelpCommand` that is not implemented (and doesn't make sense) for any other help command.  We should be providing a consistent HelpCommand interface to consumers.